File: Algoritmos.java/mdc.java
public class mdc {
    public static int calcularMDC(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return Math.abs(a);
    }

    public static void main(String[] args) {
        int numero1 = 18;
        int numero2 = 26;
        int resultado = calcularMDC(numero1, numero2);
        System.out.println("O MDC de " + numero1 + " e " + numero2 + " é: " + resultado);
    }
}

File: Algoritmos-JS/mdc.js
function mdc(a, b) {
    while (b !== 0) {
        const temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

const numero1 = 6;
const numero2 = 12;
console.log("MDC de", numero1, "e", numero2, "é", mdc(numero1, numero2)); 